Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:6.11
+
+USER root
+RUN apt-get update
+
+WORKDIR /edx/prebuilt/studio-frontend
+COPY package.json ./
+RUN npm install --silent
+
+WORKDIR /edx/src/studio-frontend
+ENTRYPOINT cp -r /edx/prebuilt/studio-frontend/node_modules /edx/src/studio-frontend && \
+    npm install && \
+    npm run start
```

Comment:
This gives us the benefit of a pre-installed `node_modules` folder on each startup, so the following `npm install` should only install new additions to `package.json` that are not yet installed into the image.